src/components/Controls.vue

```diff
@@ -69,7 +147,7 @@
 				</Actions>
 				<!-- FIXME: ActionRouter currently doesn't work as an inline action -->
 				<Actions>
-					<ActionButton icon="icon-share" @click="toggleDetailsView" />
+					<ActionButton icon="icon-settings" @click="toggleDetailsView" />
```

Comment:
Please keep such changes separate. Ideally we should use the new sidebar icon here:
![image](https://user-images.githubusercontent.com/3404133/73452617-82e71c80-436a-11ea-8494-34b8bba40381.png)